object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/framework/AutoValueSupport.java

```diff
@@ -7,12 +7,7 @@
 import com.sun.tools.javac.code.Types;
 import com.sun.tools.javac.processing.JavacProcessingEnvironment;
 import java.beans.Introspector;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.LinkedHashSet;
-import java.util.List;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
Can we get rid of this wildcard import?  You can disable wildcard imports in IntelliJ by following these instructions:

https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

Then if you do Code -> Optimize Imports in IntelliJ it will re-expand the import I think.